### PR TITLE
Update all documentation for clarity and accuracy

### PR DESCRIPTION
**Pull request by ecdye**
Additionally, switch from 80 character line length limit as it is not
necessary. Instead, follow the guideline of one sentence per line for
documentation. This allows for easy reading of git diffs when
documentation changes are made.

A few text inconsistencies were also fixed throughout the code.

Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: docs/openhabian-DEBUG.md [hunk 1]
```diff
@@ -1,227 +1,142 @@
...
-how to mount your SD card and how to modify the openHABian config file.
-Some parameters are self-explanatory but please nonetheless read the full explanation
-in the linked document.
-Given that you're already reading the debug guide, the most important parameter
-to set is likely `debugmode=maximum`.
-Once you have passed the first time boot initialization phase and you can login
-to the system, `/etc/openhabian.conf` will be used from there on. You can change
-it at any time to get output on future boot runs or if you use `openhabian-config`
-interactively.
+NOW, read [openhabian.md](https://github.com/openhab/openhabian/blob/main/docs/openhabian.md#openhabianconf)how to mount your SD card and how to modify the openHABian config file.
```
Comment (mstormi) [inline docs/openhabian-DEBUG.md:52]: a space missing

File: docs/openhabian.md [hunk 2]
```diff
@@ -1,159 +1,144 @@
...
+
+For ARM hardware that we don't support, you can try any of the [fake hardware parameters](openhabian.md/#fake-hardware-mode) to 'simulate' RPi hardware and Raspi OS.
+If that still doesn't work for you, give [Ubuntu](https://ubuntu.com/download/iot) or [ARMbian](https://www.armbian.com/) a try.
+
+Going beyond what the RPi image provides, as a manually installed set of scripts, we support running openHABian on x86 hardware on generic Debian.
 On ARM, we only support Raspberry Pi OS.
 These are what we develop and test openHABian against.
-We do **not support Ubuntu** so no promises. We provide code "as-is", it may work or not.
-Several optional components such as WireGuard or Homegear are known to expose problems.
+We do **not** actively **support Ubuntu** so no promises but we provide code "as-is" that is known to run on there.
```
Comment (mstormi) [inline docs/openhabian.md:104]: I intentionally removed the "actively". As I had to learn there's still people to insist that wuith that in they understand it means it should work to use Ubuntu (and sooner or later show up asking for help when it does not, leading this whole paragraph ad adsurdum).

**Review by mstormi — APPROVED** [review]
thanks for the footwork

**Comment by mstormi** [comment]
Would you be willing to review #1425 ?

Dunno why James Bowler retracted his one on config mgmt in Github, do you ?

**Comment by ecdye** [comment]
> Dunno why James Bowler retracted his one on config mgmt in Github, do you ?

Not sure. I hope he is just making changes then resubmitting.

**Pull request merged**